src/lib.rs

```diff
@@ -1,23 +1,33 @@
 #![warn(clippy::all)]
-#![allow(clippy::single_match)] 
+#![allow(clippy::single_match)]
 
 mod painter;
 
 pub use painter::Painter;
 
 use {
-    clipboard::ClipboardProvider,
     egui::*,
-    sdl2::{event::WindowEvent, mouse::MouseButton, mouse::SystemCursor, keyboard::{Keycode, Mod} }
+    sdl2::{
+        event::WindowEvent,
+        keyboard::{Keycode, Mod},
+        mouse::MouseButton,
+        mouse::SystemCursor,
+    },
 };
 
-use clipboard::ClipboardContext; // TODO: remove
+#[cfg(not(feature = "clipboard"))]
```

Comment:
I didn't understand this part. Does the following line (mod clipboard) compile only if clipboard feature is NOT defined? Shouldn't it be the other way around?

Reply:
Of course.